Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM valian/docker-python-opencv-ffmpeg
+
+ADD config app/config
+ADD tests app/tests
+ADD vision app/vision
+ADD run.py app/run.py
+ADD requirements.txt app/requirements.txt
+
+WORKDIR app
+
+RUN pip install -r requirements.txt
+
+CMD python run.py
```

Comment:
Add newline @ EOF

Reply:
Whoops. Why doesn't PyCharm automatically do that?